src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  --hover-color: #00acdc;
+}
+
+.container {
+  display: flex;
+  height: 60px;
+  align-items: center;
+  align-content: space-around;
   margin: 0;
 }
+
+.logo.png {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  width: 1200px;
+  justify-content: flex-end;
+  list-style: none;
+  padding: 0;
+  margin-left: 50px;
+}
+
+.nav_item {
+  display: block;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  line-height: 14px;
+  padding: 5px;
+  position: relative;
+}
+
+.nav__link.is-activ {
```

Comment:
The class `.nav__link.is-activ` seems to have a typo. If it's meant to indicate the active state, it should be `.nav__link.is-active`.

Reply:
Corrected the typo in the class .nav__link.is-active